scalarm_client_csharp/Experiment.cs

```diff
@@ -85,7 +85,16 @@ public Experiment(string experimentId, Client client) : base(client)
             Id = experimentId;
 			SimulationParamsMap = new Dictionary<ValuesMap, ValuesMap>();
         }
-
+		/// <summary>
+		/// Gets and save experiment binary files
+		/// </summary>
+		/// <returns>OK when properly saved files, error otherwise</returns>
+		/// <param name="scenarioId">Scenario identifier.</param>
+		/// <param name="path">save files path </param>
+		public string GetandSaveExperimentBinaryFiles(string scenarioId, string path)
```

Comment:
@Andriu01 - ta metoda w Experiment nie ma sensu - po to chciałem dodać do Experiment, żeby użytkownik nie musiał wpisywać experimentId (które tutaj nazwane jest niepoprawnie scenarioId)